client/app/cloud/project/compute/snapshot/add/cloud-project-compute-snapshot-add.controller.js

```diff
@@ -37,11 +37,9 @@ angular.module("managerApp")
       };
 
       self.getPriceData = function () {
-          CloudProjectComputeSnapshotPriceService.getSnapshotPrice(self.snapshotPriceStruct.size).then(function (data) {
-              self.snapshotPriceStruct.prices = data;
-
-              //We ignore the region for now and take the first total we find.  Could be improved.
-              self.snapshotPriceStruct.total = self.snapshotPriceStruct.prices[0].totalPrice;
+          CloudProjectComputeSnapshotPriceService.getSnapshotPrice(self.snapshotPriceStruct.size, self.snapshot.vm.serviceName, self.snapshot.vm.region).then(function (data) {
```

Comment:
that's a lot of parameters.

Reply:
what do you suggest ?

Reply:
some sort of DTO => snapshotInfo { size, servicename, region  }
or 
CloudProjectComputeSnapshotPriceService.getSnapshotPrice(servicename, snapshotInfo)